core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -196,8 +198,8 @@ class StageData private[spark](
 
     val rddIds: Seq[Int],
     val accumulatorUpdates: Seq[AccumulableInfo],
-    val tasks: Option[Map[Long, TaskData]],
-    val executorSummary: Option[Map[String, ExecutorStageSummary]],
+    var tasks: Option[Map[Long, TaskData]],
```

Comment:
similar here, I'd rather not modify these to be var

Reply:
Done